docs/01-overview/main-areas/observability/README.md

```diff
@@ -2,20 +2,30 @@
-- With the [Jaeger](https://github.com/jaegertracing) distributed tracing system, you can analyze the path of a request chain going through your distributed applications. This information helps you to, for example, troubleshoot your applications, or optimize the latency and performance of your solution.
+Out of the box, Kyma provides tools to collect and expose **telemetry** data, such as metrics, traces, and log data. Of course, you'll want to view and analyze the data you're collecting. This is where **observability** tools come in.
 
-## Benefits of distributed tracing
+## Collecting data
 
-Observability tools should clearly show the big picture, no matter if you're monitoring just a few or many components. In a cloud-native microservice architecture, a user request often flows through dozens of different microservices. Tools such as logging or monitoring help to track the request's path. However, they treat each component or microservice in isolation. This individual treatment results in operational issues.
+Kyma collects telemetry data with several in-cluster components:
 
-Distributed tracing charts out the transactions in cloud-native systems, helping you to understand the application behavior and relations between the frontend actions and backend implementation.
+- [Prometheus](https://prometheus.io/docs/introduction) collects metrics from Pods. Metrics are the time-stamped data that provide information on the running jobs, workload, CPU consumption, memory usage, and more. All metrics relevant for observing the in-cluster Istio Service Mesh are collected separately.
 
-The diagram shows how distributed tracing helps to track the request path.
+- [Fluent Bit](https://fluentbit.io/) collects logs.
 
-![Distributed tracing](./assets/distributed-tracing.svg)
+- Traces are sent to [Jaeger](https://www.jaegertracing.io/docs).
+
+The collected telemetry data are exposed so that you can view and analyze them with observability tools.
+
+## Analyzing data
+
+You can use the following in-cluster components to observe your applications' telemetry data:
+
+- [Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) receives and manages alerts coming from Prometheus. It can then forward the notifications about fired alerts to specific channels, such as Slack or an on-call paging system of your choice.
+- [Grafana](https://grafana.com/docs/guides/getting_started/) provides a dashboard and a graph editor to visualize metrics collected from Prometheus.
+- Kyma uses [Jaeger](https://www.jaegertracing.io/docs/) as a backend, which serves as the query mechanism for displaying information about traces.
+- Kyma uses [Loki](https://github.com/grafana/loki), which is a lightweight Prometheus-like log management system.
+- Kyma uses [Kiali](https://www.kiali.io) to enable validation, observe the Istio Service Mesh, and provide details on microservices included in the Service Mesh and connections between them.
```

Comment:
```suggestion
- Kyma uses [Prometheus](https://prometheus.io/docs/introduction), which is a lightweight backend for metrics.
- Kyma uses [Loki](https://github.com/grafana/loki), which is a lightweight Prometheus-like backend for logs.
- Kyma uses [Jaeger](https://www.jaegertracing.io/docs/) as a backend, which serves as the query mechanism for displaying information about traces.

- [Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) receives and manages alerts coming from Prometheus. It can then forward the notifications about fired alerts to specific channels, such as Slack or an on-call paging system of your choice.
- [Grafana](https://grafana.com/docs/guides/getting_started/) provides a dashboard and a query editor to visualize metrics and logs collected from Prometheus and Loki.
- Kyma uses [Kiali](https://www.kiali.io) to enable validation, observe the Istio Service Mesh, and provide details on microservices included in the Service Mesh and connections between them.
```

I mainly added prometheus here as it is a collector and a backend at the same time